packages/scripts/src/lib/app-data/postProcess/assets.ts

```diff
@@ -34,27 +35,32 @@ const APP_CORE_SIZE_KB = {
   total: APP_CORE_BUILD_KB + APP_CORE_ASSETS_KB,
 };
 
+interface IAssetFolderMetadata {
+  /** Whether this folder contains remote assets (asset pack) */
+  remote?: boolean;
+  /** Name of the folder (used for remote asset pack output directory) */
+  folderName?: string;
+}
```

Comment:
nit(blocking): the deployment config already defines
```
 /** gdrive id from end of url */
      id: string;
      /** local name for download */
      name: string;
      /** if true, the assets from this folder are treated as an asset pack to be uploaded to remote storage */
      remote?: boolean;
```
Adding another type here is confusing, as really anything that needs to be definied should be defined in the deployment config either within the data source config (currently nested in google drive), or within the remote_assets config. This should be passed to the processor directly from the workflow